.pre-commit-config.yaml

```diff
@@ -18,3 +18,23 @@
   hooks:
   - id: pydocstyle
     exclude: blog/migrations
+
+- repo: git://github.com/pre-commit/mirrors-pylint.git
+  rev: v1.9.1
+  hooks:
+  - id: pylint
+    args:
+    - --msg-template={path}:L {line:3d}:({symbol}) {msg} (C {column:2d}), ::{msg_id}
+    - --output-format=colorized
+    - |-
+        --init-hook=
+        import os, platform, sys;
+        sys.path.insert(
+          0,
+          "".join([os.getenv("HOME"), "/virtualenv/python", platform.python_version(), "/site-packages"])
+          if os.getenv("TRAVIS") else
+          "{}/test_venv/lib/python3.6/site-packages".format(os.getcwd())
```

Comment:
Do you use `test_venv` in your local machine for a virtual env or not? If it's something else, be sure to change this folder name.